pom.xml

```diff
@@ -5,14 +5,13 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>mate-academy</groupId>
-    <artifactId>oop-example</artifactId>
+    <artifactId>read-from-file</artifactId>
```

Comment:
Why do you change the artifact name?

Reply:
Nie podejmuję się wytłumaczenia, co zaszło w mojej głowie, że to zrobiłem, bo sam tego nie rozumiem ;) Przywróciłem mam nadzieję właściwą wersję.